tests/integration_tests/test_query_loop_integration.py

```diff
@@ -86,7 +86,7 @@ def test_query_loop_integration_with_one_iteration_with_not_released_child_query
         query = f"EXECUTE SCRIPT {schema_name}.AAF_RUN_QUERY_HANDLER('{args}')"
         result = conn.execute(textwrap.dedent(query)).fetchall()
     assert "E-AAF-4: Error occurred while calling the query handler." in caught_exception.value.message and \
-           "RuntimeError: Child contexts are not released" in caught_exception.value.message
+           "Following child contexts were not released:" in caught_exception.value.message
```

Comment:
```suggestion
           "The following child contexts were not released:" in caught_exception.value.message
```
You might want to add a mitigation tip here.

Reply:
There is only one Mitigation, does it help to add something like this,

Mitigation: Close all your query_handler_context